### DISCUSSION TITLE
Multiple spec files

### DISCUSSION THREAD

**Discussion by simoami** [stoplightio/elements#1761] [Q&A]
Suppose I have multiple microservices and an individual spec file for each microservice. say `orders.oas.yaml`, `cart.oas.yaml` and`catalog.oas.yaml`. Is it possible to have a single deployment of the api documentation and a way to switch between the files / services to view their api specs?

**Comment by mnaumanali94** [comment]
You can do that by using Stoplight platform and the elements dev portal. That supports multiple files and markdown. 

https://meta.stoplight.io/docs/elements/ZG9jOjEyMDU2Njc2-introduction-to-elements-dev-portal
